README.md

```diff
@@ -9,17 +9,17 @@ This web site, code and contents are not supported by or endorsed by Fox ESS. An
    </br>
 </h2>
 
-This site contains code and information for integrating Fox ESS inverters with Home Assistant. It is a refactored and evolved version, derrived from https://github.com/StealthChesnut/HA-FoxESS-Modbus.
+This site contains code and information for integrating Fox ESS inverters with Home Assistant. It is a refactored and evolved version, derived from https://github.com/StealthChesnut/HA-FoxESS-Modbus.
 
 The changes from the main branch include:
 
-* Simplified configuration: fewer lines are required in your configuration.yaml with modbus, sensors, templates and utility_meter integrations split into separate include files
+* Simplified configuration: fewer lines are required in your configuration.yaml with Modbus, sensors, templates and utility_meter integrations split into separate include files
```

Comment:
Sorry, missed this one - these are literals, probably should be quited. 'modbus' is the name of the included section in configuration.yaml, not 'Modbus'.